src/routes/tools/extreme-weather/_constants.js

```diff
@@ -107,12 +107,13 @@ export const DOY_DESCRIPTION = `<p>This can be any day of the year you wish to s
 
 export const EXTREMES_DESCRIPTION = `<h5>High Extremes</h5>
```

Comment:
@mukhtyar just remember that it's not good for a11y to use heading level elements to size text. Since it's about the same size as the regular `<p>` you could probably just do `<p><strong>High Extremes</strong></p>` for each of these.

Reply:
Fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/131/commits/b3890f612caa393a20aa0de797609c6311a64703